msg/telemetry_status.msg

```diff
@@ -1,18 +1,41 @@
-uint64 timestamp			# time since system start (microseconds)
+uint8 LINK_TYPE_GENERIC = 0
```

Comment:
Do these enums need to be in sync with any MAVLink definition? If yes could it be mentioned in a comment?